backend/routes/ratings.js

```diff
@@ -33,9 +33,32 @@ router.get("/app/:appId", function(req, res) {
   }
 });
 
-router.get("/", function(req, res) {
+router.get("/", function(_, res) {
```

Comment:
Just curious, what's the reason for replacing `req` with `_`?

Reply:
In the past I've typically used and seen others use '_' for unused params so I prob just changed it instinctively -- I'd like to encourage everyone to do this too but LMK if you have other suggestions =)